.github/workflows/pull_requests.yml

```diff
@@ -31,8 +33,8 @@ jobs:
       - name: Checkout submodules
         run: git submodule update --init
       - name: Build helm operator image
-        run: make docker-build IMG=registry.localhost:5000/sumologic/sumologic-collection-helm-operator:latest
+        run: make docker-build IMG=$IMG
```

Comment:
```suggestion
        run: make docker-build IMG="${IMG}"
```

Reply:
Same for `Makefile` :sweat_smile: which is not checked by shellcheck

Reply:
`Makefile` was mostly generated by operator-sdk and its style will not pass tests by shellcheck in many places. I fixed style in modified lines, for the rest I will prepare a new pull request.